packages/frontend/src/features/sqlRunner/components/Header/HeaderView.tsx

```diff
@@ -121,51 +121,60 @@ export const HeaderView: FC = () => {
-                                        <MantineIcon
-                                            icon={IconTrash}
-                                            color="red"
-                                        />
-                                    }
-                                    color="red"
-                                    disabled={
-                                        !(canManageSqlRunner && canManageChart)
-                                    }
-                                    onClick={() =>
-                                        dispatch(
-                                            toggleModal('deleteChartModal'),
-                                        )
-                                    }
-                                >
-                                    Delete
-                                </Menu.Item>
-                            </Menu.Dropdown>
-                        </Menu>
+                        {canManageChart && (
+                            <Menu
+                                position="bottom"
+                                withArrow
+                                withinPortal
+                                shadow="md"
+                                width={200}
+                            >
+                                <Menu.Target>
+                                    <ActionIcon variant="default">
+                                        <MantineIcon icon={IconDots} />
+                                    </ActionIcon>
+                                </Menu.Target>
+                                <Menu.Dropdown>
+                                    <Menu.Label>Manage</Menu.Label>
+                                    <Menu.Item
+                                        icon={
+                                            <MantineIcon
+                                                icon={IconLayoutGridAdd}
+                                            />
+                                        }
+                                        onClick={() =>
+                                            dispatch(
+                                                toggleModal('addToDashboard'),
+                                            )
+                                        }
+                                    >
+                                        Add to dashboard
+                                    </Menu.Item>
+                                    <Menu.Item
+                                        icon={
+                                            <MantineIcon
+                                                icon={IconTrash}
+                                                color="red"
+                                            />
+                                        }
+                                        color="red"
+                                        disabled={
+                                            !(
```

Comment:
not sure if we need one of these booleans tho. Maybe just `canManageSqlRunner`?

Reply:
done 👍